conf/reef/rbd/4-node-cluster-with-1-client.yaml

```diff
@@ -39,3 +39,5 @@ globals:
       node4:
         role:
           - client
+        no-of-volumes: 3
```

Comment:
why we need volumes on client node?
please add comment.

Reply:
Added that for testing purpose. Removing it now.